src/modules/offer/offer.controller.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  Body,
+  Controller,
+  Get,
+  Param,
+  Post,
+  Request,
+  UseGuards,
+} from '@nestjs/common';
+import { JwtAuthGuard } from 'src/modules/auth/guards/jwt-auth.guard';
+import { searchParamDto } from 'src/modules/chat-room/dto/search-param.dto';
+import { AuthRequest } from 'src/utils/@types/AuthRequest';
+import { OfferService } from 'src/modules/offer/offer.service';
+import { Offer } from 'src/modules/offer/entities/offer.entity';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Can you take either this comment or the unused variable out?

Reply:
this is some eslint bug, but it shows that is unused variable, despite the fact that I'm using it
![Screenshot from 2022-09-27 09-07-29](https://user-images.githubusercontent.com/92886035/192446806-5818933b-c819-447e-b8d4-4023d4c9392e.png)